packages/mobile/src/escrow/saga.ts

```diff
@@ -138,7 +138,7 @@ export function* transferToEscrow(action: EscrowTransferPaymentAction) {
 
     // Tranfser the funds to the Escrow contract.
     Logger.debug(TAG + '@transferToEscrow', 'Transfering to escrow')
-    yield call(registerStandbyTransaction, context, amount.toString(), escrowWrapper.address)
+    yield call(registerStandbyTransactionLegacy, context, amount.toString(), escrowWrapper.address)
```

Comment:
I think the non-legacy code also uses this. Maybe we can register both legacy and non-legacy here? So invites also show up in the standby state.

Reply:
This is done in the next PR: https://github.com/valora-inc/wallet/pull/1669